packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,6 +16,34 @@ import patch from '../patch'
 import readChunk from '../_readStreamChunk'
 import { generateToken } from '../utils'
 
+// networks field contains:
+// - <n>/ip <=> <n>/ipv4/0
+// - <n>/ipv4/<m>
+// - <n>/ipv6/<m>
+// where n is the VIF order and m is the IP order
```

Comment:
What do *VIF order* and *IP order* mean?

Reply:
As i understand, the VIF creation order. I'm open to additional information about it.